docs/resources/job.md

```diff
@@ -226,6 +226,11 @@ You can invoke Spark submit tasks only on new clusters. **In the `new_cluster` s
 
 You also need to include a `git_source` block to configure the repository that contains the dbt project.
 
+### run_job_task Configuration Block
+
+* `job_id` - (Required) ID of the job
+* `job_parameters` - (Optional) Job parameters for the task
```

Comment:
We need to specify what type this and previous parameters are

Reply:
Done